plugins/content/emailcloak/emailcloak.php

```diff
@@ -430,12 +430,12 @@ protected function _cloak(&$text, &$params)
 		 * <img src="..." title="email@example.org"> or <input type="text" placeholder="email@example.org">
 		 * The '<[^<]*>(*SKIP)(*F)|' trick is used to exclude this kind of occurrences
 		 */
-		$pattern = '~<[^<]*>(*SKIP)(*F)|' . $searchEmail . '~i';
+		$pattern = '~<[^<]*(?<!\/)>(*SKIP)(*F)|(<\w.*\"'.$searchEmail.'\".*\/\>)~i';
```

Comment:
```suggestion
		$pattern = '~<[^<]*(?<!\/)>(*SKIP)(*F)|(<\w.*\"' . $searchEmail . '\".*\/\>)~i';
```
Please fix code style error. Thanks in advance.

Reply:
i have fix the code style error. Thanks